cmd/kubelet/app/options/options.go

```diff
@@ -162,6 +162,8 @@ type KubeletFlags struct {
 	ExitOnLockContention bool
 	// seccompProfileRoot is the directory path for seccomp profiles.
 	SeccompProfileRoot string
+	// seccompDefaultProfile is the default profile to be used
```

Comment:
Is this the path to the profile, or the serialized profile json itself? Make it clear.